app/qmlV2/component/MMInput.qml

```diff
@@ -15,14 +15,16 @@ import "../Style.js" as Style
 Item {
   id: control
 
-  enum Type { Normal, Password, Search, Calendar, Scan, CopyButton }
+  enum Type { Normal, Password, Search, Calendar, Scan, CopyButton, SendButton }
```

Comment:
Hi @iiLubos, I am afraid this is not a good way forward. We need to have standalone inputs for each of the types.

Reply:
Refactored in this PR